### PR TITLE
Release v1.35.0

### PR DESCRIPTION
**Pull request by angrybayblade**
## Release summary

Version number: [e.g. 1.0.1]

## Release details

Plugins:
- Adds support for multiple transaction builders on `flashbots` ledger plugin #643 
- Pins #645 
  - `eth-account`to `>=0.5.9,<0.6.0`
  - `protobuf` to `==3.19.5`
  - `web3` to `==5.31.4`
  - `construct` to `<=2.10.61`

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] Lint and unit tests pass locally and in CI
- [ ] I have checked the fingerprint hashes are correct by running (`aea packages lock --check`)
- [ ] I have regenerated the latest API docs
- [ ] I built the documentation and updated it with the latest changes
- [ ] I have added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `aea/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest`  
      (check the READMEs of [`aea-develop`](../develop-image/README.md#publish) 
      and [`aea-user`](../user-image/README.md#publish))
- [ ] I have pushed the latest packages to the registry.
- [ ] I have uploaded the latest `aea` to PyPI.
- [ ] I have uploaded the latest plugins to PyPI.

## Further comments

Write here any other comment about the release, if any.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Release History - open AEA
 
+## 1.35.0 (2023-06-20)
+
+Plugins:
+- Adds support for multiple transaction builders on `flashbots` ledger plugin
+- Pins
+  - `eth-account`to `>=0.5.9,<0.6.0`
```
Comment (dvilelaf) [inline HISTORY.md:8]: The eth_account version with the bytes32 fix is 0.6.1, so this will be a problem.

Comment (dvilelaf) [reply HISTORY.md:8]: See [here](https://github.com/ethereum/eth-account/issues/90#issuecomment-1050317881)

Comment (0xArdi) [reply HISTORY.md:8]: flashbots doesnt support that version

**Review by 0xArdi — APPROVED** [review]

File: HISTORY.md (hunk 1 above)
Comment (dvilelaf) [reply HISTORY.md:8]: Ok, I'll keep the custom version on governatooorr for now. Viraj, please dont close the open issue then.

**Review by dvilelaf — APPROVED** [review]

**Pull request merged**